src/en/clouds-manual.md

```diff
@@ -0,0 +1,102 @@
+Title: Adding a manual cloud
+
+# Using a "Manual" cloud
+
+As well as many other types of cloud, public and private, Juju caters for the 
+case where you may not be able to access one of these traditional clouds in a 
+straightforward way; maybe you can't create additional instances, or perhaps 
+your cloud is really a collection of disparate hardware.
+
+Whatever the case, as long as Juju can log into these machines (and you have 
+sudo permissions on them) they can be used as part of a Juju cloud. It won't
+be _quite_ the same as using a standard cloud - without the ability to create
+new instances when they are desired, you will be missing out on some of the 
+Juju magic. You can still deploy and manage services thoguh, with a bit of extra
+effort.
+
+## Prerequisites for using a manual cloud:
+
+  - You will need at least two machines
+  - The machines must already have Ubuntu installed
+  - They must be running SSH
+  - You will need a login with sudo privileges on all machines
+  - All machines must be able to communicate with each other over the network
+
+## Bootstrapping the cloud
+
+As with other clouds, Juju needs to create a controller to manage models and 
+them other instances in the cloud. Bootstrapping is the same as for 'regular' 
+clouds, except in this case, after the cloud type you should supply the network 
+address of the machine you wish to use as the controller:
+
+```bash
+juju bootstrap mycloud manual/192.168.1.128
+```
+
+Note that it is also possible to use the other common bootstrap parameters here.
+
+## Adding machines to the cloud
+
+Bootstrapping the cloud created the controller and initial models that Juju
+uses. Deploying services will require additional machines to be made available
+though. The more conventional Juju cloud types can create machines
+automatically, but in the case of a manual cloud you will have to specify these
+resources.
+
+This is done by using the `add-machine` command in Juju. This will install the 
+default series on a given resource (in this case a machine on the network to
+which you have access and the relevant permissions), and prepare it for
```

Comment:
i wonder if we need to repeat the requirements, it was mentioned in the beginning